$MirOS: ports/mailnews/pine/patches/patch-imap_src_dmail_dmail_c,v 1.2 2005/05/04 23:37:39 tg Exp $
--- imap/src/dmail/dmail.c.orig	Tue Sep  2 21:09:48 2003
+++ imap/src/dmail/dmail.c	Tue Dec  9 13:01:48 2008
@@ -33,7 +33,7 @@ extern int errno;		/* just in case */
 
 /* Globals */
 
-char *version = "2003(12)";	/* dmail release version */
+char *version = "2003(12)L";	/* dmail release version */
 int debug = NIL;		/* debugging (don't fork) */
 int flagseen = NIL;		/* flag message as seen */
 int trycreate = NIL;		/* flag saying gotta create before appending */
@@ -208,7 +208,7 @@ int deliver (FILE *f,unsigned long msgle
 				/* have a mailbox specifier? */
   if (mailbox = strchr (user,'+')) {
     *mailbox++ = '\0';		/* yes, tie off user name */
-    if (!*mailbox || !strcmp ("INBOX",ucase (strcpy (tmp,mailbox))))
+    if (!*mailbox || !compare_cstring ((unsigned char *) mailbox,"INBOX"))
       mailbox = NIL;		/* user+ and user+INBOX same as user */
   }
   if (!*user) user = myusername ();
